etc/portage/sets/eessi-2022.11-linux-x86_64

```diff
@@ -0,0 +1,19 @@
+app-editors/emacs
+app-editors/vim
+app-shells/bash-completion
+dev-python/GitPython
+dev-python/keyring
+dev-python/pip
+dev-python/pycodestyle
+dev-python/pyyaml
+dev-python/rich
+dev-util/patchelf
+media-fonts/dejavu
+media-fonts/liberation-fonts
+=sys-apps/archspec-0.1.3::eessi
+sys-apps/dbus
+sys-cluster/lmod
+sys-cluster/rdma-core
+=sys-cluster/reframe-3.9.1::eessi
+=sys-fabric/opa-psm2-11.2.205::eessi
```

Comment:
latest version is 11.2.230, we should update to that?

Reply:
Added ebuild for 11.2.230.